modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -19,6 +19,7 @@ class Representative < ApplicationRecord
 
       validates :poa_codes, presence: true
 
+      # rubocop:disable Lint/DuplicateBranch
```

Comment:
Seems like the branch this is referring to actually is screwy. Might not be part of this PR, but probably worth refactoring.